File: FinesModel4/Models/AlumnoComision.cs
using System;
using System.Collections.Generic;

namespace FinesModel4.Models;

public partial class AlumnoComision
{
    public string Id { get; set; } = null!;

    public DateTime Creado { get; set; }

    public bool? Activo { get; set; }

    public string? Observaciones { get; set; }

    public string? Comision { get; set; }

    public string Alumno { get; set; } = null!;

    public string? Estado { get; set; }

    public uint? Pfid { get; set; }

    public virtual Alumno AlumnoNavigation { get; set; } = null!;

    public virtual Comision? ComisionNavigation { get; set; }
}

File: Data/AlumnoComision.php
<?php

namespace Fines2;

use SqlOrganize\Sql\Entity;
use Exception;
use DateTime;

class AlumnoComision extends Entity
{

    public function __construct()
    {
        $this->_entityName = "alumno_comision";
        $this->_db = \SqlOrganize\Sql\DbMy::getInstance();
        $this->setDefault();
    }

    /** @var int|null */
    public ?int $activo = null;

    /** @var string|null */
    public ?string $alumno = null;

    /** @var string|null */
    public ?string $comision = null;

    /** @var DateTime|null */
    public ?DateTime $creado = null;

    /** @var string|null */
    public ?string $estado = null;

    /** @var string|null */
    public ?string $id = null;

    /** @var string|null */
    public ?string $observaciones = null;

    /** @var int|null */
    public ?int $pfid = null;

    /** @var Alumno|null (fk alumno_comision.alumno _m:o alumno.id) */
    public ?\Fines2\Alumno $alumno_ = null;

    /** @var Comision|null (fk alumno_comision.comision _m:o comision.id) */
    public ?\Fines2\Comision $comision_ = null;

}
